.github/workflows/go.yml

```diff
@@ -58,7 +58,8 @@ jobs:
 
     - name: Deploy Metal LB Operator
       run: |
-        IMG=${built_image} KUSTOMIZE_DEPLOY_DIR="config/kind-ci/" make deploy
+        make deploy-cert-manager
```

Comment:
nit on the commit message: remove the "attempts". Hopefully, when it lands it will enable the webhooks :-)